src/translations/en.json

```diff
@@ -15,6 +15,13 @@
 		"current-c-ratio": "current c-ratio",
 		"target-c-ratio": "target c-ratio",
 		"transaction": "Transaction",
+		"transaction-abbreviation": "TX",
+		"burn": "BURN",
+		"claim": "CLAIM",
+		"mint": "MINT",
+		"pending": "PENDING",
+		"complete": "COMPLETE",
+		"view": "VIEW",
```

Comment:
We usually do not uppercase in the translation file, just use an uppercase text transform on the css rules